pom.xml

```diff
@@ -0,0 +1,135 @@
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>nl.aerius</groupId>
+  <artifactId>sld-generator-maven-plugin</artifactId>
+  <name>SLD Generator Maven Plugin</name>
+  <description>AERIUS SLD Generator Maven Plugin generates SLD style files from JSON.</description>
+  <version>1.0-SNAPSHOT</version>
+  <packaging>maven-plugin</packaging>
+
+  <organization>
+    <name>Ministry of Economic Affairs of The Netherlands</name>
+    <url>https://www.government.nl/ministries/ez</url>
+  </organization>
+
+  <licenses>
+    <license>
+      <name>GNU Affero General Public License</name>
+      <url>http://www.gnu.org/licenses/agpl-3.0.txt</url>
+    </license>
+  </licenses>
+
+  <properties>
+    <java.version>11</java.version>
+    <maven-plugin.version>3.6.0</maven-plugin.version>
+    <junit.version>5.7.2</junit.version>
+    <jackson.version>2.12.3</jackson.version>
+  </properties>
+
+  <distributionManagement>
+    <repository>
+      <id>aerius-nexus</id>
+      <url>https://nexus.aerius.nl/repository/maven-releases/</url>
+    </repository>
+    <snapshotRepository>
+      <id>aerius-nexus</id>
+      <url>https://nexus.aerius.nl/repository/maven-snapshots/</url>
+    </snapshotRepository>
+  </distributionManagement>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.apache.maven</groupId>
+      <artifactId>maven-plugin-api</artifactId>
+      <version>${maven-plugin.version}</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.maven.plugin-tools</groupId>
+      <artifactId>maven-plugin-annotations</artifactId>
+      <version>${maven-plugin.version}</version>
+      <scope>provided</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-text</artifactId>
+      <version>1.9</version>
+    </dependency>
+
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <version>${jackson.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-engine</artifactId>
+      <version>${junit.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-params</artifactId>
+      <version>${junit.version}</version>
+      <scope>test</scope>
+    </dependency>
+  </dependencies>
+
+  <repositories>
+    <repository>
+      <id>aerius-nexus-public</id>
+      <name>AERIUS Nexus repository</name>
+      <url>https://nexus.aerius.nl/repository/maven-public/</url>
+    </repository>
+    <repository>
+      <id>OSGEO</id>
+      <name>OSGEO GeoServer Maven Repository</name>
+      <url>https://repo.osgeo.org/repository/release/</url>
+    </repository>
```

Comment:
Is the OSGEO repository needed?

Reply:
No, not needed. I used the pom of geoserver-core as basis, that's why it was here in the first place. Removed.